rpc_util.go

```diff
@@ -502,4 +502,12 @@ const (
 // Version is the current grpc version.
 const Version = "1.9.0-dev"
 
+// These constants specify the minimum and maximum acceptable RPC version values.
+const (
+	MaxRPCVersionMajor = 2
```

Comment:
Actually, after more deliberation on our side, could you name these:

```go
ProtocolVersionMaxMajor
ProtocolVersionMaxMinor
ProtocolVersionMinMajor
ProtocolVersionMaxMajor
```

And also please update the comment above to change "RPC version" to "protocol version"?

1. I prefer to name consts/vars/etc in a hierarchical fashion, putting the most significant parts first, which in this case would be ProtocolVersion, Major/Minor, Min/Max.  (This makes them sort better, and effectively namespaces them.)
1. We thought "RPC version" as a name was potentially confusing, and gets worse when it's `grpc.RPCVersion`.

Thanks!

Reply:
That's a good point. Consts are renamed.